src/components/settings/index.tsx

```
@@ -0,0 +1,99 @@
+    const settingsNav = getSettingsNav(t)
+
+    return (
+      settingsNav
+        // Get all items
+        .map((group) => group.items)
+        // Flatten them
+        .flat()
+        // Make object in shape as SettingsTabsObject type
+        .reduce(
+          (prevValue, value) => ({
+            ...prevValue,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)